letsencrypt/cli.py

```diff
@@ -871,6 +879,30 @@ def _renewal_conf_files(config):
     return glob.glob(os.path.join(config.renewal_configs_dir, "*.conf"))
 
 
+def _renew_describe_results(renew_successes, renew_failures, parse_failures):
```

Comment:
This function completely controls the output of `letsencrypt renew --dry-run`, so it should print slightly different messages in that case.